src/main/java/com/ksabbak/javaserver/Main.java

```diff
@@ -6,14 +6,29 @@
     public static void main(String[] args) throws IOException {
 
         int portNumber = 5000;
-        Server server = new Server();
 
         ServerSocket serverSocket =
                 new ServerSocket(portNumber);
         while (true){
 
-            try (Socket socket = serverSocket.accept()) {
-                String httpResponse = "HTTP/1.1 " + server.statusCode() + "\r\n\r\n";
+            try (Socket socket = serverSocket.accept();
+                 BufferedReader in = new BufferedReader(new InputStreamReader(socket.getInputStream()))) {
+
+                String unparsedHeader = "";
+                String line;
+                Boolean crlf = false;
```

Comment:
Abbreviations like CRLF can be unclear. Recommend expanding this variable name to `emptyLine` or something like that.